source/sass/buyers-guide/views/product.scss

```diff
@@ -11,6 +11,27 @@ $pni-product-breakpoint-larger: $bp-md;
   .seal-of-approval {
     @include seal-of-approval(1rem, 0, 76px, 76px, 140px, 140px);
   }
+
+  .product-header-content.show-privacy-ding {
+    position: relative;
+
+    &:after {
+      content: "";
+      position: absolute;
+      top: 24px;
+      right: 24px;
+      background: url(../_images/buyers-guide/icon-privacy-ding.svg) center /
```

Comment:
We have a band that reads `Warning: *privacy not included with this product` already so I'm treating this badge as purely decorative image and therefore I implemented it as a pseudo element.

Reply:
yep, makes sense to me

Reply:
Note that localized versions of this image are coming (see https://github.com/mozilla/foundation.mozilla.org/issues/5260) and that it’s probably easier to handle if the image is set in the template.
We can still play with the attribute selector `html[lang="fr"]`, tough